src/goto-symex/symex_other.cpp

```diff
@@ -128,7 +131,7 @@ void goto_symext::symex_other(
           statement==ID_array_replace)
   {
     // array_copy and array_replace take two pointers (to arrays); we need to:
-    // 1. dereference the pointers (via clean_expr)
+    // 1. dereference pointers within the pointers (via clean_expr)
```

Comment:
How about just `remove any dereference expressions`, since we're no longer creating a top-level special one?

Reply:
Thanks, done!